src/App.jsx

```diff
@@ -1,11 +1,11 @@
 import axios from "axios";
 import Cookies from "js-cookie"; // i realize we are using both react-cookie and js-cookie
 import React, { useState } from "react";
-import { CookiesProvider } from "react-cookie";
```

Comment:
I put cookies provider in index.js on the outer app level

Reply:
okie